### PR TITLE
add some convenient method

### PR DESCRIPTION
**Pull request by jackpan123**

### CODE REVIEW

File: src/main/java/com/amihaiemil/docker/ExecInstance.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import java.io.IOException;
+import javax.json.JsonObject;
+
+/**
+ * Exec Api
+ * @version 1.00 2019-12-24
+ * @author Jack Pan
+ * */
+public interface ExecInstance {
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/ExecInstance.java:11]: @jackpan123 We already have the execs implemented. A Docker exec is represented by the ``Exec`` interface, we don't need this ``ExecInstance``.

File: src/main/java/com/amihaiemil/docker/Containers.java [hunk 2]
```diff
@@ -95,10 +95,17 @@ Container create(
...
      * @return Docker.
      */
     Docker docker();
+
+    /**
+     * Get this container.<br><br>
+     * @param containerId of the Container
+     * @return this container.
+     */
+    Container get(final String containerId);
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/Containers.java:110]: @jackpan123 Please make a separate PR for this ``get`` method and also write some tests for it.

File: src/main/java/com/amihaiemil/docker/Exec.java [hunk 3]
```diff
@@ -46,4 +46,16 @@
...
+     * Return low-level information about this exec instance.
+     * @param containerId
+     * @param exec param of object
+     * @return ExecInstance object.
+     * @see <a href="https://docs.docker.com/engine/api/v1.35/#operation/ImageInspect">Inspect Image</a>
+     * @throws IOException If something goes wrong.
+     * @throws UnexpectedResponseException If the status response is not
+     *  the expected one (200 OK).
+     */
+    ExecInstance create(final String containerId, final JsonObject exec) throws IOException, UnexpectedResponseException;
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/Exec.java:59]: @jackpan123 I'm pretty sure this method should be in the [Execs](https://github.com/amihaiemil/docker-java-api/blob/master/src/main/java/com/amihaiemil/docker/Execs.java) interface.

File: src/main/java/com/amihaiemil/docker/RtContainers.java [hunk 4]
```diff
@@ -158,4 +158,22 @@ HttpClient client() {
...
+    /**
+     * Get this container.<br><br>
+     *
+     * @param containerId of the Container
+     * @return this container.
+     */
+    @Override
+    public Container get(String containerId) {
+        return new RtContainer(
+            Json.createObjectBuilder().build(),
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/RtContainers.java:171]: @jackpan123 You could add the id to the JSON body.

File: src/main/java/com/amihaiemil/docker/Exec.java (hunk 3 above)
Comment (jackpan123) [reply src/main/java/com/amihaiemil/docker/Exec.java:59]: Only get method in Execs interface,  create method can provide execute the command line in a certain container.

Comment (jackpan123) [reply src/main/java/com/amihaiemil/docker/Exec.java:59]: Could I change the return value of the method?

**Comment by 0crat** [comment]
Job `gh:amihaiemil/docker-java-api#347` is not assigned, can't get performer

**Comment by 0crat** [comment]
Pull request [#347](https://github.com/amihaiemil/docker-java-api/issues/347) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)
